boards/remote/Makefile.include

```diff
@@ -0,0 +1,51 @@
+# Define the cpu used by the Re-mote board
+export CPU       =  cc2538
+export CPU_MODEL ?= cc2538sf53
+
+# Define the default port depending on the host OS
+OS := $(shell uname)
+ifeq ($(OS),Linux)
+  PORT ?= /dev/ttyUSB1
+else ifeq ($(OS),Darwin)
+  PORT ?= $(shell ls -1 /dev/tty.usbserial-* | head -n 1)
+else
+  $(info CAUTION: No flash tool for your host system found!)
+  # TODO: add support for windows as host platform
+endif
+export PORT
```

Comment:
You can simplify this section with 
```
PORT_LINUX ?= /dev/ttyUSB1
PORT_DARWIN ?= $(shell ls -1 /dev/tty.usbserial-* | head -n 1)
```
have a look at e.g. the `boards/stm32f4discovery/Makefile.include`...